kct/deploy_kip7_token_contract/src/main/java/Boilerplate.java

```diff
@@ -0,0 +1,98 @@
+ */
+public class Boilerplate {
+    // You can directly input values for the variables below, or you can enter values in the caver-java-examples/.env file.
+    private static String nodeApiUrl = ""; // e.g. "https://node-api.klaytnapi.com/v1/klaytn";
+    private static String accessKeyId = ""; // e.g. "KASK1LVNO498YT6KJQFUPY8S";
+    private static String secretAccessKey = ""; // e.g. "aP/reVYHXqjw3EtQrMuJP4A3/hOb69TjnBT3ePKG";
+    private static String chainId = ""; // e.g. "1001" or "8217";
+    private static String deployerAddress = ""; // e.g. "0xf5d3322418f2f2257d9255ad4a7ff1d50312d438"
+    private static String deployerPrivateKey = ""; // e.g. "0x42f6375b608c2572fadb2ed9fd78c5c456ca3aa860c43192ad910c3269727fc7"
+    private static String recipientAddress = ""; // e.g. "0xeb709d59954f4cdc6b6f3bfcd8d531887b7bd199"
+
+    public static void main(String[] args) {
+        try {
+            loadEnv();
+            run();
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+    }
+
+    public static String objectToString(Object value) throws JsonProcessingException {
+        ObjectWriter ow = new ObjectMapper().writer().withDefaultPrettyPrinter();
+        return ow.writeValueAsString(value);
+    }
+
+    public static void loadEnv() {
+        Dotenv env;
+        String workingDirectory = System.getProperty("user.dir");
+        Path workingDirectoryPath = Paths.get(workingDirectory);
+        String projectRootDirectory = "caver-java-examples";
+        String currentDirectoryName = workingDirectoryPath.getName(workingDirectoryPath.getNameCount() - 1).toString();
+        String envDirectory = currentDirectoryName.equals(projectRootDirectory) ?
+                workingDirectoryPath.toString() :
+                workingDirectoryPath.getParent().getParent().toString();
+
+        // Read `/path/to/caver-java-examples/.env` file.
+        env = Dotenv.configure().directory(envDirectory).load();
+
+        nodeApiUrl = nodeApiUrl.equals("") ? env.get("NODE_API_URL") : nodeApiUrl;
+        accessKeyId = accessKeyId.equals("") ? env.get("ACCESS_KEY_ID") : accessKeyId;
+        secretAccessKey = secretAccessKey.equals("") ? env.get("SECRET_ACCESS_KEY") : secretAccessKey;
+        chainId = chainId.equals("") ? env.get("CHAIN_ID") : chainId;
+        deployerAddress = deployerAddress.equals("") ? env.get("DEPLOYER_ADDRESS") : deployerAddress;
+        deployerPrivateKey = deployerPrivateKey.equals("") ? env.get("DEPLOYER_PRIVATE_KEY") : deployerPrivateKey;
+        recipientAddress = recipientAddress.equals("") ? env.get("RECIPIENT_ADDRESS") : recipientAddress;
+    }
+
+    public static void run() throws Exception {
+        HttpService httpService = new HttpService(nodeApiUrl);
+        httpService.addHeader("Authorization", Credentials.basic(accessKeyId, secretAccessKey));
+        httpService.addHeader("x-chain-id", chainId);
+        Caver caver = new Caver(httpService);
+
```

Comment:
I think it is better to add a comment that deployer address must have KLAY enough to deploy contract.

Reply:
That will be great.
I'll update that feedback to this PR soon.

Reply:
@sirano11 
Updated it [here](https://github.com/klaytn/caver-java-examples/pull/6/commits/170ba4ed0d007dc45485e1275b28a856b0ca2850) :)

Reply:
@jimni1222 
Hi jasmine. 
Can I make a PR at `caver-js-examples` applying this feedback after this PR merged